src/Lib/PHPManagerConfiguration.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace PHPManager\PHPManager\Lib;
+
+final readonly class PHPManagerConfiguration
+{
+    function __construct(public string $phpManagerDirectory = '/.php-manager', public string $distDirectory = '/dist')
```

Comment:
Proposal: istead of `distDirectory=dist`, you could have `distRelativeDirectory=dist`, which would make it clearer that it is relative to `phpManagerDirectory`. Then we could construct `distDirectory=$distDirectory/$distRelativeDirectory`. Likewise we could have a `phpDir`, `buildDir` and `composerDir` constructed the same way. Although at this point might be worth extracting another feat PR to introduce the configuration.

Entirely personal, when there is this kind of logic, I tend to prefer to put it in another factory rather than the constructor, to keep the ability to create the object without any extra processing (either for perf reasons or for convenience in tests), e.g.:

```php
readonly class Configuration {
  function __construct(
    public string $phpManagerDir,
    public string $distDirectory,
  ) {}

  static function create(
    string $phpManagerDir,
    string $distRelativeDir,
  ) {
    return new self(
      $phpManagerDir,
      $phpManagerDir.'/'.$distRelativeDir,
    );
  }
}
```

Reply:
Dist is currently outside of the php manager dir. It does make sense to move it inside, then we can fix this... But it'll require changing the way the installer works, I'll leave that for another PR.